README.md

```diff
@@ -0,0 +1,93 @@
+# Atlas Basic Terraform Module
+
+This Terraform module creates a basic setup on [MongoDB Atlas] [atlas]. 
+
+It creates the following resources:
+
+- A MongoDB Atlas project.
+- A MongoDB Atlas cluster provisioned with AWS, GCP or AZURE provider. It can also be a free tier (TENANT) cluster.
+- Zero or more IP addresses.
+- Zero or more CIDR blocks.
+- One or more database users.
+
+You can find detailed information of the module's input and output variables in the Terraform Public Registry <LINK_TO_REGISTRY>
```

Comment:
This is a missing link. Should it be this way?

Reply:
For now yes, once we publish the module on the public registry we will have the correct link.